examples/tutorials/sklean_models.ipynb

```diff
@@ -0,0 +1,325 @@
+{
```

Comment:
<p>Yes, you can indeed. Nevertheless, let's simplify the tutorial and make it easier to grasp and re-use. So, define pipelines that work with one model and the pass the model via config. Thus you'll shorten pipelines and the whole tutorial without compromising its functionality</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>